raco/backends/myria/myria.py

```diff
@@ -1268,7 +1268,16 @@ def fire(self, exp):
             return MyriaLimit(exp.count,
                               algebra.Collect(
                                   MyriaLimit(exp.count, exp.input)))
+        return exp
+
+
+class GlobalOrderBy(rules.Rule):
 
+    def fire(self, exp):
+        if exp.__class__ == algebra.OrderBy:
+            return MyriaInMemoryOrderBy(
```

Comment:
If we add a local OrderBy, the Collect operator will gather tuples in no particular order, right? I'm not sure I understand how this would help. I can certainly add it though.